packages/tooltip/src/Tooltip.js

```diff
@@ -5,15 +5,43 @@ import values from "object.values";
 import { Container, StyledTooltip } from "./styledComponents/Tooltip";
 import alignments from "./constants/alignments";
 
-const Tooltip = props => {
-  const content = props.children || props.text;
-  return (
-    <Container>
-      {content && <StyledTooltip {...props}>{content}</StyledTooltip>}
-    </Container>
-  );
+export const getAutoAlign = bodyWidth => containerRef => align => {
```

Comment:
On the `Tooltip` class we could have access to all these properties with no need to create this closure...
Maybe it would be simplified it it was on the class instance.
What you think? Any particular reasons for it to live here?

Reply:
I made it like this for testability. So it's not necessary to inject the `containerRef` mock into the component nor manipulate the `document` global.

Reply:
Hmmm.... Still think we do not need a closure here since we always pass the body width and tests can be done on component did mount, but ok.